go/event_test.go

```diff
@@ -0,0 +1,65 @@
+package testGoSDK
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	xenapi "github.com/xenserver/xenserver-samples/go/goSDK"
+)
+
+var eventTypes []string
+var maxTries = 3
+var timeout = 10.0
+var token = ""
+
+func TestEventNext(t *testing.T) {
+	eventTypes = append(eventTypes, "*")
+	err := xenapi.Event.Register(session, eventTypes)
```

Comment:
The calls event.register and event.unregister are deprecated. See http://xapi-project.github.io/xen-api/classes/event.html. (They were needed with the old event.next, but not with event.from).

Reply:
Basically the whole TestEventNext is not needed as this mechanism was deprecated back in XS 6.0.

Reply:
OK, if we don't expect to use those API anymore, I'll remove it from all test case. Thanks.